src/Honeybee_addHBGlz.py

```diff
@@ -147,6 +147,12 @@ def addPotentialChildSurface(HBSurface):
                         return
     
                 # add it to the base surface
+                if(HBSurface.BC== "Adiabatic"):
```

Comment:
What about Ground or Surface boundary conditions? Wouldn't it just be easier to write:
```
 if HBSurface.BC != "Outdoors":
```

Reply:
I think it should be only checking:
` if HBSurface.BC != "Surface":`
We want to keep interior Surface same, but changing "Ground" and "Adiabatic" to "Outdoor".